README.md

```diff
@@ -2,15 +2,18 @@
 *Bot lists must meet these minimum requirements to be considered a bot list.*
 
 ---
-
-
-1. Your bot list must have a respectable TLD. (.tk does not count)
-2. Your bot list can **NOT** be hosted on glitch.
-3. Your bot list should not display all the listed bots on the front page.
-4. Your bot list must be usable. No broken links/buttons.
-5. Your bot list must have consistent uptime. (Minor downtime for fixing a problem is fine)
-
-
+Your bot list...
+1. must have a respectable TLD.
+   - .tk TLDs are not allowed.
+   - No subdomains used from non-botlists/free hosting (e.g. Glitch.me)
+2. can **NOT** be hosted on glitch.
+3. should not display all the listed bots on the front page.
+4. must be usable. No broken links/buttons.
+   - It should work on all commonly used browsers (Chrome, Opera and yes even Edge)
+5. must have consistent uptime.
+   - Minor downtime due to fixes or maintenance are ok.
+6. should have an easy to understand API-Documentation, if an API is available.
+7. can **NOT** be a clone of an already existing (open source) bot list.
```

Comment:
Incorrect. Let’s start a new issue or PR to discuss this further. Remove from this one :)

Reply:
What is incorrect?
Having a (one on one) clone is imo not a good way to make a bot list since you basically only mirror the original.

Reply:
Create a separate issue or PR to discuss this further and remove it from here. We allow clones in some situations.